.styleci.yml

```diff
@@ -0,0 +1,8 @@
+enabled:
+
+disabled:
+  - align_double_arrow
```

Comment:
but leads to lots of unnecessary diffs when changing code. does symfony code style say something about this? i would stick to that so we can avoid arguing here :-)